momentum/4/config-options-summary.md

```diff
@@ -11,11 +11,12 @@ If an option has a default value, it is shown in the `Default` column followed b
 
 The `Version` column indicated the version(s) of Momentum that support the option.
 
-<a name="all-options-table"></a> 
+<a name="all-options-table"></a>
 
 
 | Option/Description | Type | Default | Version | Scopes |
 | --- | --- | --- | --- | --- |
+| [prefer_ipv6_mxs](/momentum/4/config/ref-prefer_ipv6_mxs) - Prefer IPv6 addresses over IPv4 addresses at a given MX priority level | sending | false | 4.0 and later | global |
```

Comment:
everything else in this list looks to be in alphabetic order. Can you move this down to follow that setup?